idl/github.com/uber/cadence/shared.thrift

```diff
@@ -930,3 +930,30 @@ struct DescribeWorkflowExecutionResponse {
   10: optional WorkflowExecutionConfiguration executionConfiguration
   20: optional WorkflowExecutionInfo workflowExecutionInfo
 }
+
+struct GetPollerHistoryRequest {
+  10: optional string domain
+  20: optional TaskList taskList
+  30: optional TaskListType taskListType
+}
+
+struct GetPollerHistoryResponse {
+  10: optional list<PollerInfo> pollers
+}
+
+enum TaskListType {
```

Comment:
Can we use TaskType here? We are going to use TaskListType for the type of task list like REGULAR, STICKY, and HOST_SPECIFIC.

Reply:
at least right now, the task_list_type is already used in cassandra: 

-- Stores activity or workflow tasks
CREATE TABLE tasks (
  domain_id        uuid,
  task_list_name   text,
  task_list_type   int, -- enum TaskListType {ActivityTask, DecisionTask}
  type             int, -- enum rowType {Task, TaskList}
  task_id          bigint,  -- unique identifier for tasks, monotonically increasing
  range_id         bigint, -- Used to ensure that only one process can write to the table
  task             frozen<task>,
  task_list        frozen<task_list>,
  PRIMARY KEY ((domain_id, task_list_name, task_list_type), type, task_id)
) WITH COMPACTION = {
    'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy'
  };